### PR TITLE
remove old promql editor

### PR DESCRIPTION
**Pull request by Nexucis**
We removed the old promql editor in Prometheus (https://github.com/prometheus/prometheus/pull/9452). I think it makes it is removed here as well :).

Signed-off-by: Augustin Husson <husson.augustin@gmail.com>

### CODE REVIEW

**Comment by Nexucis** [comment]
@onprem ;)

File: pkg/ui/react-app/src/pages/graph/ExpressionInput.test.tsx [hunk 1]
```diff
@@ -1,28 +1,15 @@
...
-    key,
-    nativeEvent: {},
-    preventDefault: () => {
-      // Do nothing.
-    },
-  } as React.KeyboardEvent<HTMLInputElement>);
-
-describe('ExpressionInput', () => {
-  const metricNames = ['instance:node_cpu_utilisation:rate1m', 'node_cpu_guest_seconds_total', 'node_cpu_seconds_total'];
+describe('CMExpressionInput', () => {
```
Comment (onprem) [inline pkg/ui/react-app/src/pages/graph/ExpressionInput.test.tsx:8]: Just a nit, for staying consistent with component name.
```suggestion
describe('ExpressionInput', () => {
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -36,6 +36,10 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 - [#4792](https://github.com/thanos-io/thanos/pull/4792) Store: Fix data race in BucketedBytes pool.
 - [#4769](https://github.com/thanos-io/thanos/pull/4769) Query-frontend+api: add "X-Request-ID" field and other fields to start call log.
 
+### Changed
+
+- [#4864](https://github.com/thanos-io/thanos/pull/4864) Remove the old PromQL editor
```
Comment (onprem) [inline CHANGELOG.md:41]:
```suggestion
- [#4864](https://github.com/thanos-io/thanos/pull/4864) UI: Remove the old PromQL editor
```

It'd be nice if we can prefix this with `UI: `. More context for people reading changelog.

**Review by onprem — APPROVED** [review]
Looking good :100:. Just some nits.

**Comment by Nexucis** [comment]
thanks for your review @onprem. I updated the PR, it should be good now :)

**Review by onprem — APPROVED** [review]
:rocket:

**Pull request merged**